src/tests/gov/nasa/jpf/test/java/net/URLClassLoaderTest.java

```diff
@@ -519,4 +519,17 @@ public void testThrownException() throws ClassNotFoundException, MalformedURLExc
     }
     movePkgBack();
   }
+
+  @Test
+  public void getPackageInfo() throws MalformedURLException {
+    movePkgOut();
+     if(verifyNoPropertyViolation())  {
+       URL[] urls = { new URL(dirUrl) };
+       TestClassLoader cl = new TestClassLoader(urls);
+         assertNotNull(cl.getDefinedPackage("java.lang"));
+         assertNotNull(cl.getDefinedPackage("java.lang").getName());
```

Comment:
change to `assertEquals` with the package name as the expected value

Reply:
changed.